Automattic/sensei
# Add/import file form

### Changes proposed in this Pull Request

* Implements the uploading page for import.

### Testing instructions

* The API enpoints are working and the frontend is connected to them however since the import is not running yet, there are not many observable results.
* Navigate to Sensei LMS > Import.
* Try uploading files and click continue.
* A job is created when the first file is uploaded and it is started once continue is pressed.
* Once the job has started, no more files can be uploaded (this is a nice way to test the error messages for each file).
* In case the API call when clicking 'Continue' fails, an error message is displayed on the left of the 'Continue' button. To test this you can close the server before pressing continue (the error message is not descriptive, it is returned by `apiFetch`)
* If you want to delete the started job to retest uploading, you can use this query: `delete from wp_options where option_name like 'sensei-data-port%';`. The files uploaded will still live on the uploads directory though.
